features/telemetry/adapters/netflow/src/main/java/org/opennms/netmgt/telemetry/adapters/netflow/v5/fields/InetAddressField.java

```diff
@@ -0,0 +1,53 @@
+/*******************************************************************************
+ * This file is part of OpenNMS(R).
+ *
+ * Copyright (C) 2017-2017 The OpenNMS Group, Inc.
+ * OpenNMS(R) is Copyright (C) 1999-2017 The OpenNMS Group, Inc.
+ *
+ * OpenNMS(R) is a registered trademark of The OpenNMS Group, Inc.
+ *
+ * OpenNMS(R) is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as published
+ * by the Free Software Foundation, either version 3 of the License,
+ * or (at your option) any later version.
+ *
+ * OpenNMS(R) is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with OpenNMS(R).  If not, see:
+ *      http://www.gnu.org/licenses/
+ *
+ * For more information contact:
+ *     OpenNMS(R) Licensing <license@opennms.org>
+ *     http://www.opennms.org/
+ *     http://www.opennms.com/
+ *******************************************************************************/
+
+package org.opennms.netmgt.telemetry.adapters.netflow.v5.fields;
+
+import java.net.InetAddress;
+import java.net.UnknownHostException;
+import java.nio.ByteBuffer;
+
+public class InetAddressField extends AbstractField<String> {
+
+    public InetAddressField(int startIndex, int endIndex, ByteBuffer data, int offset) {
+        super(startIndex, endIndex, data, offset);
+    }
+
+    @Override
+    protected String extractValue() {
+        byte[] bytes = getFieldBytes();
+        if (bytes == null ||bytes.length != 4) {
+            throw new IllegalStateException("Could not parse field to InetAddress");
+        }
+        try {
+            return InetAddress.getByAddress(bytes).toString().replaceFirst("/", "");
```

Comment:
Can't we just use `InetAddressUtils` here?

Reply:
Originally the idea was to not have any dependencies to a opennms library. That way we could release the "flow package parsing" as it's own project and have others benefit from the work. If we don't want to do that, in fact we should use `InetAddressUtils` instead of replicating that logic here.